src/rest/api-response.spec.js

```diff
@@ -0,0 +1,29 @@
+import ApiResponse from './api-response.js';
+
+const response = new ApiResponse(200, { type: 'Fiat', model: '500', color: 'white' });
+
+describe('apiResponse', () => {
+  it('returns true when statuses are the same', () => {
```

Comment:
Just a note, maybe add extra spaces between tests. I think it might improve readability

Reply:
If prettier didn't fix it, then there is no point to do so.